lib/eco/export/cpython.py

```diff
@@ -59,6 +61,30 @@ def _run(self):
         self.tm.export_as_text(f[1])
         return subprocess.Popen(["python2", f[1]], stdout=subprocess.PIPE, stderr=subprocess.STDOUT, bufsize=0)
 
+    def _debug(self):
+        f = tempfile.mkstemp(suffix='.py')        
+        code = self.tm.export_as_text(f[1])
+
```

Comment:
I think it would be useful here (or wherever it makes more sense) to check if the user has `remote_pdb` installed, and if not throw a warning. Otherwise the user gets a traceback.